src/assets/scss/styles.scss

```diff
@@ -0,0 +1,445 @@
+@import "./stylesheet";
+@import "./reset";
+
+$t-primary: #181919;
+
+@import "./_header";
+@import "./_modal";
+@import "./_footer";
+
+html {
+  font-family: "Montserrat";
+  scroll-behavior: smooth;
+}
+*,
+*:before,
+*:after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+.container {
+  width: 1170px;
+  margin: 0 auto;
+  padding: 0 15px;
+}
+@media (max-width: 1169px) {
+  .container {
+    width: 1024px;
+  }
+}
+@media (max-width: 1023px) {
+  .container {
+    width: 900px;
+  }
+}
+@media (max-width: 899px) {
+  .container {
+    width: 768px;
+  }
+}
+@media (max-width: 767px) {
+  .container {
+    max-width: calc(100% - 30px);
+    padding: 0;
+  }
+}
+
+/*------------------- section about-------------------*/
+
+.about {
+  padding: 125px 0 165px 0;
+  text-align: center;
+  background-color: #f8fafa;
+  &__title {
+    font-size: 38px;
+    color: $t-primary;
+    font-weight: bold;
+    padding-bottom: 5px;
+    text-transform: uppercase;
+    letter-spacing: 1.5px;
+  }
+  &__text {
+    color: #5e5f5f;
+    font-family: "Merriweather";
+    font-size: 18px;
+    text-transform: uppercase;
+    line-height: 1.5;
+    letter-spacing: 1.3px;
+  }
+  &__content {
+    padding-top: 116px;
+    display: flex;
+    justify-content: space-around;
+  }
+  &__img {
+    margin-left: 33px;
+  }
+}
+
+.description {
+  width: 490px;
+  text-align: left;
+  padding-top: 29px;
+  margin-left: 85px;
+
+  &__title {
+    font-family: "Merriweather";
+    font-size: 20px;
+    font-weight: bold;
+    text-transform: uppercase;
+    padding-bottom: 23px;
+    letter-spacing: 1.4px;
+  }
+  &__text {
+    font-family: "Raleway";
+    font-size: 16px;
+    line-height: 1.75;
+    padding-bottom: 27px;
+    letter-spacing: 0.8px;
+    color: #929393;
+  }
+  &__text:nth-of-type(2) {
+    padding-bottom: 34px;
+  }
```

Comment:
In this case it is better to create modificator (BEM).

Reply:
Thanks, fixed it.